### PR TITLE
PR to make ANSIBLE_AI_ENABLE_PLAYBOOK_ENDPOINT default to True

### PR DESCRIPTION
**Pull request by justjais**
Jira Issue: <https://issues.redhat.com/browse/AAP-35817>

## Description

PR to make ANSIBLE_AI_ENABLE_PLAYBOOK_ENDPOINT default to True, as per the discussion

### Steps to test
1. Pull down the PR
2. ...
3. ...

### Scenarios tested

NA

## Production deployment

- [x] This code change is ready for production on its own
- [ ] This code change requires the following considerations before going to production:

### CODE REVIEW

File: ansible_ai_connect/main/settings/base.py [hunk 1]
```diff
@@ -622,10 +622,7 @@ def is_ssl_enabled(value: str) -> bool:
 # Playbook Generation/Explanation endpoints
 # ------------------------------------------
 ANSIBLE_AI_ENABLE_PLAYBOOK_ENDPOINT = (
-    os.getenv(
-        "ANSIBLE_AI_ENABLE_PLAYBOOK_ENDPOINT", "False" if DEPLOYMENT_MODE == "on-prem" else "True"
-    ).lower()
-    == "true"
+    True
```
Comment (jameswnl) [inline ansible_ai_connect/main/settings/base.py:625]: we don't want to hardcode this

Comment (jameswnl) [reply ansible_ai_connect/main/settings/base.py:625]: we still want it to be able to take env setting

**Review by hasys — APPROVED** [review]
Looks good to me, thank you Sumit

File: ansible_ai_connect/main/settings/base.py (hunk 1 above)
Comment (justjais) [reply ansible_ai_connect/main/settings/base.py]: @hasys yea, updated the same

**Pull request merged**